app/models/concerns/project_scopes.rb

```diff
@@ -48,6 +48,6 @@ module ProjectScopes
                  .having('count(*) >= ?', tags.split.flatten.length)
     }
     scope :with_analysis, -> { active.where.not(best_analysis_id: nil) }
-    scope :active_enlistments, -> { active.joins(:enlistments).where(enlistments: { deleted: false }) }
+    scope :active_enlistments, -> { active.joins(:enlistments) }
```

Comment:
Removed the where clause because enlistment is having default scope with the same one.

So adding the where clause was adding two times the same conditions.